src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier COLOR_SUPPLIER;
+    private static final Random RANDOM;
+
+    static {
+        COLOR_SUPPLIER = new ColorSupplier();
+        RANDOM = new Random();
+    }
+
+    public Ball getRandomBall() {
+        int number = RANDOM.nextInt(100);
```

Comment:
All magic numbers in your code should be constants. https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants